WBooks/Components/ReviewTable/ReviewViewController.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  ReviewViewController.swift
+//  WBooks
+//
+//  Created by ana.mancuso on 17/10/2022.
+//
+
+import UIKit
+
+class ReviewViewController: UIViewController {
+
+    //MARK: Properties
+    
+    private lazy var reviewView = ReviewView()
+    private let reviewViewModel: ReviewViewModel
+    
+    //MARK: Initialization
+    
+    init (reviewViewModel: ReviewViewModel) {
+        self.reviewViewModel = reviewViewModel
+        super.init(nibName: nil, bundle: nil)
+    }
+    
+    required init? (coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        reviewView.tableView.delegate = self
+        reviewView.tableView.dataSource = self
+        reviewView.tableView.registerCell(cellType: ReviewTableViewCell.self)
+        reviewViewModel.loadSampleLibrarys() {
+        self.reviewView.tableView.reloadData()
+        }
+    }
+    
+    override func loadView() {
+        view = reviewView
+    }
+    
+}
+extension ReviewViewController: UITableViewDelegate, UITableViewDataSource {
+    
+    // MARK: - Table view data source
+    
+    func numberOfSections(in tableView: UITableView) -> Int {
+        return 1
+    }
```

Comment:
I think there is no need to use it in this case, since the default value is 1, this function could be removed